<ui version="4.0" >
 <class>SelectionDialog</class>
 <widget class="QDialog" name="SelectionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="uiTitle" >
     <property name="text" >
      <string>&lt;center>&lt;b>Choose a plug-in&lt;/b>&lt;/center></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="uiList" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushAccept" >
       <property name="text" >
        <string>&amp;Accept</string>
       </property>
       <property name="icon" >
        <iconset resource="common.qrc" >:/images/images/ok.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="common.qrc" >:/images/images/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="common.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushCancel</sender>
   <signal>clicked()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>352</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel" >
     <x>370</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pushAccept</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>153</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel" >
     <x>259</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
